code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -876,7 +887,7 @@ var/global/list/robot_modules = list(
 		"Cooler Master" =  list(ROBOT_CHASSIS = "coolermaster_syndi", ROBOT_PANEL = "coolermaster_syndi", ROBOT_EYES = "coolermaster"),
 		"Phage" =          list(ROBOT_CHASSIS = "phage_syndi", ROBOT_PANEL = "phage_syndi", ROBOT_EYES = "phage")
 	)
-
+	all_access = TRUE
```

Comment:
note that this is the syndiborg/antag borg. it should definitely not have all access.

Reply:
I am just adding what it already had, which it already had all access before. It is 35TC item, expensive, not having any access might make it more useless.